src/AvaloniaMath.Example/App.xaml.cs

```diff
@@ -9,5 +12,18 @@ public override void Initialize()
         {
             AvaloniaXamlLoader.Load(this);
         }
+
+        public override void OnFrameworkInitializationCompleted()
+        {
+            if (ApplicationLifetime is IClassicDesktopStyleApplicationLifetime desktop)
```

Comment:
Why do you need this type check here? Are you not sure if the lifetime is always of this type?

Reply:
No, I didn't think about that at all. It's just that this code is common in the template

https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/21fbd4cf2b940801609c7d32f66a14f4f2d97403/templates/csharp/app/App.axaml.cs#L14-L22

Reply:
Alright.